components/listing/Listing.tsx

```diff
@@ -22,17 +23,17 @@ const Listing: React.FC<ListingProps> = ({ tasks = [] }) => {
       <div className="mt-6 grid grid-cols-1 row-gap-8 bg-orange-600">
         <div
           id="projects-listing-section"
-          className="border-t border-gray-200 pt-8 grid grid-cols-1 row-gap-6 lg:grid-cols-4 lg:gap-5"
+          className="border-t border-gray-200 pt-8 grid grid-cols-1 row-gap-6 lg:grid-cols-3md lg:gap-5 xl:grid-cols-4"
         >
-          <div>
+          <div className="xs: text-center sm:text-left">
```

Comment:
to działa z tą spacją? `xs: text-center`

Reply:
tak działa- ale już usunąłem. Przy kolejnym commicie wyleci ta spacja.